.make/Makefile.lnx

```diff
@@ -12,6 +12,9 @@ GOCOVMERGE_BIN=$(VENDOR_DIR)/github.com/wadey/gocovmerge/gocovmerge
 DOCKER_BIN_NAME=docker
 DOCKER_COMPOSE_BIN_NAME=docker-compose
 
+GOLINT_BIN=$(VENDOR_DIR)/github.com/golang/lint/golint/golint
```

Comment:
Not that it is required, but I'd suggest to add these lines with .exe suffix to Makefile.win as well.